linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_nss_db/rule.yml

```diff
@@ -40,3 +40,9 @@ ocil: |-
     run the following command:
     <pre>$ pkcs11-switch</pre>
     The output should return <pre>opensc</pre>
+
+warnings:
+    - general: |-
+       NSS modules information are stored in NSS database which is in binary format. Currently
+       it is not possible to check NSS database using OVAL. This is the reason there is no OVAL
+       check for this rule.
```

Comment:
Hello @marcusburghardt  and thank you for the update. Please just add a new line here and the PR is good to be merged. Thank you.

Reply:
Done. Thanks